sydney/index.html

```diff
@@ -5,26 +5,75 @@
 city_page: true
 city: Sydney
 short: SYD
+gmaps: syd-2018-map.js
 header_link: /sydney
 title_override: true
 title: UNIHACK Sydney
-description: UNIHACK is Sydney's premier student hackathon, coming back soon!
+description: UNIHACK is Sydney's premier student hackathon, running in August at Coder Academy. Students get 24 hours to design and build something awesome – an app, website, piece of hardware... anything goes. Solve a problem, impress us and win some prizes!
 ---
 
 <div id="event-head">
   <div class="event-head-inner">
     <div class="unihack-wrap container-fluid">
       <div class="row">
         <div class="col-md-12">
-          <h2>Hey Sydney 👋</h2>
-          <p>We’re coming back August 4 – 5!</p>
+          <h2>
+            <span>UNIHACK Sydney is back.</span>
+            <span>Get excited for August 4!</span>
+          </h2>
+          <p>Our best hackathon yet is coming soon – grab your tickets now!</p>
           <p>
-            <a class="uh_btn" href="https://eventbrite.com.au/e/unihack-2018-sydney-hackathon-tickets-45706344908">Early Bird tickets</a>
+            <a class="uh_btn" href="https://unihack2018syd.eventbrite.com.au/?aff=website">Early Bird Tickets</a>
           </p>
         </div>
       </div>
     </div>
   </div>
 </div>
 
+{% include event/about.html %}
+
+<div class="event-block no-border" id="venue">
+  <div class="unihack-wrap container-fluid">
+    <div class="event-location-block">
+      <h3>Venue</h3>
+      <p class="address">Coder Academy Sydney</p>
+      <p class="address">7 Kelly Street</p>
+      <p class="address">Ultimo, NSW 2007</p>
+
+      <h4>How do I get there?</h4>
+      <p>
+        Coder Academy is a short walk from Sydney Central station. Take exit 7 then walk down
+        Broadway and Wattle Street, until you arrive at Kelly Street.
+      </p>
+      <p>
+        The venue is also located a short walk from busses that service Broadway Shopping Centre,
+        as well as a short walk from Light Rail services.
+      </p>
+    </div>
+  </div>
+  <div class="google-maps" id="syd-2018-map"></div>
+</div>
+
+<div class="page-block event-tweet event-quote no-border">
+  <div class="unihack-wrap container-fluid">
+    <div class="row justify-content-md-center">
+      <div class="col-md-12">
+        <i class="icon fa fa-twitter"></i>
+        <p class="tweet">
+          Absolutely love hearing all the ideas at <a href="https://twitter.com/unihacksyd/">@unihacksyd</a> !! Excited to hear who the winners are 🎉
```

Comment:
is the space `</a> !!` intentional?

Reply:
Ahh it is! From the tweet: https://twitter.com/j_ennychu/status/899130527256412160

Reply:
Sorry I didn't realise it's a quote